docs/modules/ROOT/pages/how-tos/testing/integration/creating.adoc

```diff
@@ -1 +1,180 @@
+      description: Placeholder task that prints the Snapshot and outputs standard TEST_OUTPUT
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+      taskSpec:
+        params:
+        - name: SNAPSHOT
+        results:
+        - name: TEST_OUTPUT
+          description: Test output
+        steps:
+        - image: registry.redhat.io/openshift4/ose-cli:latest
+          env:
+          - name: SNAPSHOT
+            value: $(params.SNAPSHOT)
+          script: |
+            dnf -y install jq
+            echo -e "Example test task for the Snapshot:\n ${SNAPSHOT}"
+            // Run custom tests for the given Snapshot here
+            // After the tests finish, record the overall result in the RESULT variable
+            RESULT="SUCCESS"
+
+            // Output the standardized TEST_OUTPUT result in JSON form
+            TEST_OUTPUT=$(jq -rc --arg date $(date +%s) --arg RESULT "${RESULT}" --null-input \
+              '{result: $RESULT, timestamp: $date, failures: 0, successes: 1, warnings: 0}')
+            echo -n "${TEST_OUTPUT}" | tee $(results.TEST_OUTPUT.path)
+----
+
+. Add your new custom test as an integration test in {ProductName}.
+
+.. For additional instructions on adding an integration test, see Adding an integration test.
+
+.Data injected into the PipelineRun of the integration test
+
+When you create a custom integration test, {ProductName} automatically adds certain parameters, workspaces, and labels to the PipelineRun of the integration test. This section explains what those parameters, workspaces, and labels are, and how they can help you.
+
+Parameters:
+
+* *`SNAPSHOT`*: contains the snapshot of the whole application as a JSON string. This JSON string provides useful information about the test, such as which components {ProductName} is testing, and what git repository and commit {ProductName} is using to build those components. For information about snapshot JSON string, see link:https://github.com/konflux-ci/integration-examples/blob/main/examples/snapshot_json_string_example[an example snapshot JSON string].
+
+Labels:
+
+* *`appstudio.openshift.io/application`*: contains the name of the application.
+
+* *`appstudio.openshift.io/component`*: contains the name of the component.
+
+* *`appstudio.openshift.io/snapshot`*: contains the name of the snapshot.
+
+* *`test.appstudio.openshift.io/optional`*: contains the optional flag, which specifies whether or not components must pass the integration test before deployment. 
+
+* *`test.appstudio.openshift.io/scenario`*: contains the name of the integration test (this label ends with "scenario," because each test is technically a custom resource called an `IntegrationTestScenario`). 
+
+
+.Verification
+
+After adding the integration test to an application, you need to trigger a new build of its components to make {ProductName} run the integration test. Make a commit to the GitHub repositories of your components to trigger a new build.
```

Comment:
@arewm Could you please help with the possible ways? Do we have some source I can refer to? If we need to add some new content, I'd prefer we can close this one out and create a new Jira in the backlog.

Reply:
@arewm As we already have [this PR](https://github.com/konflux-ci/docs/pull/29), once merged I'll add a note here so that a user can refer to it on ways to start a new build. Does that sound good?

Reply:
Sounds great, thanks.